config/src/test/java/org/springframework/security/config/http/OAuth2ResourceServerBeanDefinitionParserTests.java

```diff
@@ -235,25 +235,26 @@ public void getWhenBearerTokenBeforeNotBeforeThenInvalidToken() throws Exception
 	}
 
 	@Test
-	public void getWhenBearerTokenInTwoPlacesThenInvalidRequest() throws Exception {
-		this.spring.configLocations(xml("JwkSetUri")).autowire();
+	public void getWhenBearerTokenInTwoPlacesAndQueryParameterNotSupportedThenAcceptsRequest() throws Exception {
```

Comment:
Because this is a `GET`, I think we should be able to restore the old test if we add a new check to the `isParameterTokenSupportedForRequest()` method. See below.